examples/src/MenuStructure.tsx

```diff
@@ -46,6 +55,34 @@ export const mainMenu: StateSpace<UIFragment, UIFragmentSpec, {}> = [
             main: MoreMenusAnt,
         },
     },
+    {
+        key: 'navRefMagic',
+        label: 'NavRef tricks',
+        subStates: [
+            {
+                key: 'simpleJump',
+                label: 'Simple jump',
+                displayOnly: true,
+                // Here, we are using this NavRef to address a different part of the app.
+                // (This menu item will jump there.)
+                // (This only works if this schema has been attached to somewhere in the menu tree.)
+                navRefCall: memTableRef.call({ groupId: 'foo', objectId: 'William8' }),
+            },
+            {
+                key: 'encodedJump',
+                label: 'Encoded jump',
+                displayOnly: true,
+                // Here, we are going to use an encoded link, which stores all in info belonging to
+                // a NavRef call, without knowing anything about the path.
+                pathTokens: ['redirect', memTableRef.call({ groupId: 'bar', objectId: 'Mary1' }).toString()],
```

Comment:
I presume that the `redirect` pathToken is a special token. Will the system warn us, if'd try to manually redefine this token?

Reply:
No, redirect is not special.